scripts/ci/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ev
+
+: "${BUILD_STAGINGDIRECTORY:?BUILD_STAGINGDIRECTORY environment variable not set}"
+: "${BUILD_SOURCESDIRECTORY:=`cd $(dirname $0); cd ../../../; pwd`}"
```

Comment:
I've found using cd and pwd to establish a working directory to be fragile. I prefer using BASH_SOURCE and relative paths:
``` bash
"$(dirname ${BASH_SOURCE[0])/../../.."
```